imblearn/utils/tests/test_docstring.py

```diff
@@ -55,6 +67,11 @@ def __init__(self, param_1, param_2):
         self.param_2 = param_2
 
 
+if sys.version_info.minor == "13":
```

Comment:
That doesn't work. It returns `False`. How about `sys.version_info >= (3, 13)` since it will be needed for future versions as well?

Reply:
That's true that we need it for future python version. I falsely trust copilot without checking that it returns an integer (and we test for alpha python version). Thanks I'll make the fix.

Reply:
Or, if you want to stick with checking the minor version number only, compare to an integer, since that is what `sys.version_info.minor` returns. But that will only work until 3.14 is released.

Reply:
Thanks! Appreciated.

Btw, my last comment was not a response. I didn't get to see your comment to my first comment until I refreshed the page. :disappointed:

Reply:
OK, I uploaded again the new wheels. Sorry for this.

Reply:
No worries and thanks again for the quick fix.